backend/tracim_backend/models/data.py

```diff
@@ -1183,6 +1184,7 @@ def get_children(self, recursively: bool = False) -> ["Content"]:
             children.append(child)
             if recursively:
                 children.extend(child.get_children(recursively=recursively))
+        children = sorted(children, key=lambda child: child.content_id)
```

Comment:
same thing here: we should sort by content_id, and by content_revision_id if content_id is the same

Reply:
same answers : we sort content.